targets/drive/mobile/scripts/configure-share-extension.js

```diff
@@ -0,0 +1,49 @@
+const fs = require('fs')
+const path = require('path')
+const rootdir = process.env.PWD
+const installWithPluginId = 'com.web-mystery.cordova.openwith-ios'
+
+module.exports = function (context) {
+  if (context.opts.cordova.plugins.indexOf(installWithPluginId) < 0) return
```

Comment:
To speed things up a bit, the script early-outs if it detects the plugin is not installed. This is the best I could come up with.

Note that it's ok if it runs multiple times anyway.